### ISSUE TITLE
Filtering?

### ISSUE THREAD

**Issue by u2giants** [audreyt/ethercalc#684]
Can Ethercalc do filtering?

**Comment by eddyparkinson** [comment]
No, or a least not an easy way. It is able to hide rows/cols, and there are ways to add a rule that would hide rows/cols that matched a rule.  

Hide rows with command(...) 
---

This has a lot of power, but is not that easy.  The audit tab shows the commands  http://sheet.cellmaster.com.au/example_command

Hope it helps.
